include/drivers/sensor.h

```diff
@@ -173,6 +176,10 @@ enum sensor_channel {
 	SENSOR_CHAN_GAUGE_DESIRED_VOLTAGE,
 	/** Desired charging current in mA */
 	SENSOR_CHAN_GAUGE_DESIRED_CHARGING_CURRENT,
+	/** Shunt voltage in uV */
+	SENSOR_CHAN_SHUNT_VOLTAGE,
+	/** Bus voltage in mV */
+	SENSOR_CHAN_BUS_VOLTAGE,
```

Comment:
As stated above, shunt voltage is probably not needed.

Existing `SENSOR_CHAN_VOLTAGE` can be used instead of defining a new sensor channel.

Reply:
Done